driver.sh

```diff
@@ -137,7 +167,7 @@ build_linux() {
     [[ $ARCH != "x86_64" ]] && cat ../configs/tt.config >> .config
     mako_reactor olddefconfig &>/dev/null
   fi
-  mako_reactor ${image_name}
+  mako_reactor
```

Comment:
Did you mean to remove this?

Reply:
IIRC, 4.14 locally failed (no such target).  Let me triple check.   Thanks for the reference to the commit; I didn't realize we were saving build time by not building default modules.

Reply:
nvm, seems to work.